contractcourt/briefcase.go

```diff
@@ -176,11 +176,6 @@ func (a ArbitratorState) String() string {
 	}
 }
 
-// resolverType is an enum that enumerates the various types of resolvers. When
-// writing resolvers to disk, we prepend this to the raw bytes stored. This
-// allows us to properly decode the resolver into the proper type.
-type resolverType uint8
```

Comment:
I think the intent of this was to mark the constants below with this type.

Reply:
Ok, left in place and added to the values